src/main/kotlin/com/terraformation/backend/report/ReportService.kt

```diff
@@ -214,7 +214,7 @@ class ReportService(
       body: LatestReportBodyModel? = null
   ): LatestReportBodyModel {
     val organization = organizationStore.fetchOneById(organizationId)
-    val isAnnual = body?.isAnnual ?: (ZonedDateTime.now(clock).minusMonths(3).quarter == 4)
+    val isAnnual = body?.isAnnual ?: (ZonedDateTime.now(clock).minusMonths(2).quarter == 4)
```

Comment:
I think this won't be quite right if this code is creating a report in March (say, for an org whose reporting was enabled late). It'll